Gemfile.lock

```diff
@@ -57,10 +57,10 @@ GIT
 
 GIT
   remote: https://github.com/department-of-veterans-affairs/vets-json-schema
-  revision: 8fcd11a03aa1e081a72a0a249ba092da6c5f064a
+  revision: b4d49a7527d7ba79665be9875acbea8c7eeae62d
```

Comment:
This will be updated with the commit hash from vets-json-schema master branch once the corresponding PR gets merged. This will remain a draft PR until then.